plugins-compat-tester/src/test/java/org/jenkins/tools/test/PluginCompatTesterTest.java

```diff
@@ -174,9 +174,24 @@ public void testMatcher() throws Throwable {
 		assertTrue("No matches",m.matches());
 		assertTrue("Invalid group", m.group(1).equals("2.7.3-milestone"));
 
+		//TODO: restore
```

Comment:
Why? Is that filename considered valid or invalid?

Reply:
My patch causes a regression. I need to talk to @raul-arabaolaza to understand the idea behind the current patterns. Maybe it's fine to allow these version formats, I see no special reasons to restrict it

Reply:
Neither do I, my experience in the PCT is not long enough to give the reason why this was originally there and I remember having to patch this in the past for similar reasons, so maybe is time to be more open-minded  regarding the possible versions of core